lib/rhc/commands/port_forward.rb

```diff
@@ -97,9 +97,14 @@ def run(app)
             end
           end
         else
-          debug "Using #{ssh_executable} to determine forwarding ports."
-          ssh_cmd = "#{ssh_executable} #{ssh_uri.user}@#{ssh_uri.host} '#{list_ports_cmd} 2>&1'"
-          status, output = run_with_system_ssh(ssh_cmd)
+          debug "\nUsing #{ssh_executable} to determine forwarding ports."
+          ssh_cmd = "#{ssh_executable} #{ssh_uri.user}@#{ssh_uri.host} '#{list_ports_cmd}'"
+          begin
+            status, output = run_with_system_ssh(ssh_cmd)
+          rescue RHC::SSHCommandFailed
+            ex_forward_cmd = "#{ssh_executable} -N -L <source_port>:<gear_ip>:<destination_port>"
```

Comment:
I'd like to, but I don't think it is possible. The command we are running will list all of the services and their ip:port combination so that we can loop through them. We don't actually have the gear service ips if this command fails.